### DISCUSSION TITLE
Text replacement for messages

### DISCUSSION THREAD

**Discussion by kieranperk** [EssentialsX/Essentials#5182] [Ideas and Feature Suggestions] [answered]
For example, if a user typed a certain character, word or phrase, that specific part of the message would be replaced with a user defined word/character/phrase.

An example of where this feature could be used is where (like hypixel) when a player types "ez", it replaces the whole message to a random funny phrase set by the server owner. Or another, which could replace ":burger:" to a specific character that could be textured by a custom resource pack to look like a burger emoji.

**Comment by Evidentsinger14** [comment, marked as answer]
There are plugins that do this. Seems pointless for essx to do it as well.

**Comment by Chew** [comment]
In addition to what Evident says, this is far too complicated to consider as well.
